generators/typescript/express/cli/package.json

```diff
@@ -30,7 +30,7 @@
   },
   "devDependencies": {
     "@fern-fern/generator-exec-sdk": "^0.0.845",
-    "@fern-fern/ir-sdk": "^43.0.0",
+    "@fern-fern/ir-sdk": "^0.0.3515",
```

Comment:
I regenerated the SDK locally with the fix in this PR. We can edit these versions to `^46.0.0` in a follow-up.